site/content/en/docs/enterprise/subscription-managment.md

```diff
@@ -14,102 +14,204 @@ you'll learn how to take control of your subscriptions and manage them.
+| **Team** | The **Team** is **for collaboration**, it removes limitations of the **Free** plan for the whole organization, allowing you to share paid benefits with your colleagues. <br><br>The **monthly payment for the plan depends on the number of team members you've added**. All limits of the **Free** plan will be removed. <Br><br> **Note**: The organization owner is also part of the team. So, if you have three annotators working, you'll need to pay for 4 seats (3 annotators + 1 organization owner). |
+
+<!--lint enable maximum-line-length-->
+
+### Annual plans
+
+Whether you're a new user, or have a subscription to **Team** or **Solo** plan, [you can
+subscribe to our annual plan](https://www.cvat.ai/post/cvat-annual-plans)
+and save up to 30% on CVAT.ai usage costs.
+
+The annual subscription offers all the benefits of our paid plans but at
+a more affordable monthly rate.
+
+For more information, see [How to switch from monthly subscription to annual?](#how-to-switch-from-monthly-subscription-to-annual)
+
 ## Billing
 
 This section describes the billing model and gives short a
 description of limitations for each plan.
 
+There are two types of subscriptions available for both the Solo
+and Team plans: monthly and annual.
+
 For more information,
 see: [Pricing Plans](https://www.cvat.ai/post/new-pricing-plans)
 
-### Solo plan
+### How to add VAT/tax and other information to the CVAT.ai invoice before the first payment?
+
+To ensure VAT (tax) information and other relevant details are included on your CVAT.ai invoices,
+it's important to add this information before making the first payment.
+
+Here’s how you can do it:
+
+1. Sign up for a CVAT.ai account and log in.
+2. (Optional) If you add the VAT/tax number to the organization,
+   first create an [organization](/docs/manual/advanced/organization/) and
+   switch to an **Organization** account.
+3. Navigate to the top right corner, next to the nickname,
+   click on the arrow > uprate to the plan.
+4. Switch on the **I would like the invoice to include additional
+   data (address, phone number, VAT information)** toggle, select the
+   best payment period for you, and click **Get Started**.
+
+![Stripe Link](/images/add-invoice.jpg)
+
+5. You will see the billing page:
+
+![Stripe Link](/images/add-vat.jpg)
+
+- Phone number **(1)**.
+- **Billing Address**: Enter the billing address you want to appear on the invoice
+     in the address field **(2)**.
+- **VAT Information** and **Business Name**: Select the checkbox **I am purchasing as a business** and enter
+     your VAT and business name information **(3)**.
+
+6. Select checkbox **I agree to refund policy** **(4)**.
```

Comment:
Probably we need to double ask the reader to read our refund policy before they buy a subscription.